src/sentry/celery.py

```diff
@@ -101,6 +101,11 @@ class SentryCelery(Celery):
     task_cls = SentryTask
 
 
+@signals.worker_process_init.connect
+def record_worker_init(*args, **kwargs):
+    metrics.incr("sentry.jobs.process.start")
```

Comment:
You may want to add a tag to identify the worker

Reply:
replied in slack, we should ahve `kube_deployment` and whatever global app tags are registered